azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -612,6 +612,7 @@ public ExecutableFlow getExecutableFlow(final int execId) {
    * delete execution dir pertaining to the given execution id
    */
   private void deleteExecutionDir(final int executionId) {
+    LOGGER.debug("Deleting execution directory for " + executionId);
```

Comment:
Would you like this output to happen *inside* the synchronized block, in case we don't enter it right away (or at all)?